x-pack/plugins/observability_solution/dataset_quality/kibana.jsonc

```diff
@@ -25,7 +25,7 @@
       "lens"
     ],
     "optionalPlugins": [],
-    "requiredBundles": ["unifiedHistogram"],
+    "requiredBundles": ["unifiedHistogram", "discover"],
```

Comment:
**nit**: as far as I see, from discover this work requires only the import of a couple of variables to use for the locators, it doesn't rely on the start contract and I think you can remove this direct dependency on discover, the optimizer should automatically detect that import and add it to the `requiredBundles`, elsewhere you can do that manually too.